hooks/pre-receive-hooks/block_unsigned_commits.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
The [example in the documentation](https://help.github.com/enterprise/2.7/admin/guides/developer-workflow/creating-a-pre-receive-hook-script/#setting-permissions-and-pushing-a-pre-receive-hook-to-github-enterprise) uses ```#!/usr/bin/env bash```.